app/res/values/styles.xml

```diff
@@ -305,4 +305,32 @@
         <item name="android:windowBackground">"@color/black"</item>
         <item name="android:navigationBarColor">@color/black</item>
     </style>
+
+        <style name="CustomButtonStyle" parent="Widget.MaterialComponents.Button">
+            <item name="background">@color/button_primary_color</item>
+            <item name="android:textColor">@color/white</item>
+            <item name="android:textSize">16sp</item>
+            <item name="android:gravity">center</item>
+            <item name="android:padding">12dp</item>
+            <item name="height">40dp</item>
+            <item name="cornerRadius">22dp</item>
+            <item name="iconPadding">8dp</item>
+            <item name="android:textAllCaps">false</item>
+            <item name="android:textStyle">bold</item>
+        </style>
+    
+    <style name="CustomButtonStyleOutline" parent="Widget.MaterialComponents.Button.OutlinedButton">
+        <item name="shapeAppearanceOverlay">@style/MyShapeAppearance</item>
+    </style>
+    <style name="MyShapeAppearance" parent="">

```

Comment:
_MyShapeAppearance_ is too generic, better to choose a more context related name